tests/test_examples.py

```diff
@@ -103,3 +103,28 @@ def test_tracking_merlion(reset_database, engine: sa.Engine, tracking_store: Sql
         assert session.query(SqlExperiment).count() == 2
         assert session.query(SqlMetric).count() == 4
         assert session.query(SqlParam).count() == 5
+
+    with engine.begin() as conn:
+        conn.execute(sa.sql.text("REFRESH TABLE doc.anomalies"))
+        conn.execute(sa.sql.text("REFRESH TABLE testdrive.runs"))
+
+        # Anomalies that have been manually annotated
+        anomalies_manual = conn.execute(sa.sql.text("SELECT * FROM doc.anomalies WHERE experiment_id IS NULL"))
+        assert anomalies_manual.rowcount == 3
+
+        # Count anomalies that have been generated by the latest run
+        anomalies_generated = conn.execute(
+            sa.sql.text(
+                """
+                SELECT *
+                FROM doc.anomalies
+                WHERE experiment_id = (
+                    SELECT experiment_id
+                    FROM testdrive.runs
+                    ORDER BY start_time DESC
+                    LIMIT 1
+                )
+                """
+            ),
+        )
+        assert anomalies_generated.rowcount >= 1
```

Comment:
I'm getting varying results here. Generates three anomalies locally, but only one here on GitHub. Not sure why, but it shouldn't matter for testing the code path of getting the anomalies saved into the database.

Reply:
Let's investigate on behalf of a subsequent iteration.